testdata/projects/simple-slow/docker-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+
+sleep 30
+touch artifacts/out.txt
+
+exit 0
```

Comment:
This directory should be here only if it's actually used in mistry's tests.

Reply:
Sure, removed.